examples/deepcopy-gen/generators/deepcopy.go

```diff
@@ -762,6 +764,8 @@ func (g *genDeepCopy) doSlice(t *types.Type, sw *generator.SnippetWriter) {
 			sw.Do("}\n", nil)
 		} else if t.Elem.Kind == types.Struct {
 			sw.Do("(*in)[i].DeepCopyInto(&(*out)[i])\n", nil)
+		} else if t.Elem.Kind == types.Alias {
```

Comment:
> thockin:
> a) this should just be an ||
> b) if it is an Alias, you still need to check the underlying Kind, no?

Reply:
If you get an alias, the alias needs to be to a struct.